docs/src/tips.md

```diff
@@ -39,3 +39,80 @@ HTML formats.
     img = plot(...)
     img = DisplayAs.Text(DisplayAs.PNG(img))
     ```
+
+### [Adding admonitions using compound line filtering](@id admonitions-md)
+
+Admonitions are a useful feature for drawing attention to particular elements of 
+documentation. They are [documented in Documenter.jl](https://juliadocs.github.io/Documenter.jl/stable/showcase/#Basic-Markdown-1) and an example of their use can be seen above in the blue 'note' box.
+They are parsed by Documenter.jl from markdown but their syntax is not parsed
+by Julia scripts or notebooks so we need to use the `#md` line filter when
+constructing admonitions from `.jl` source files sent to Literate.jl. For example:
+
+```julia
+#md # !!! note "Be aware!"
+#md #     This a note style admonition!
+```
+
+It is important to note that both `#md` and the second `#` are required. Literate.jl
+interprets the first `#md` as a markdown exclusive line, and then strips it out. The 
+second `#` tells Literate.jl that the line should be parsed as markdown and not a 
+Julia code block. If you only include `#md` and not the second `#` then it will 
+be parsed into Julia example block in the final documentation and not an actual 
+admonition.
+
+### [Custom parsing for markdown and notebook compatible admonitions](@id admonitions-compatibility)
+
+As mentioned above, admonitions are not compatible with Jupyter notebooks.
+(Though at time of writing this documentation,
+[this is an open issue in Jupter](https://github.com/jupyter/notebook/issues/1292)
```

Comment:
typo: Jupter -> Jupyter

Reply:
Thanks well spotted! I'm not sure the best way to fix it (new pull request, or maybe there is a way I can modify it here directly) will try and figure it out over the weekend.

Reply:
I think you just need to commit to the branch the PR is coming from (in this case your master branch in your fork it looks like)...

Reply:
Great that worked, I didn't know if it would create a separate pull request or not, but it has merged succesfully with the original pull request. Fixed in commit https://github.com/fredrikekre/Literate.jl/pull/115/commits/b328a4cf33ec8c45ddcd0704e8eb8a4b16a71293